benches/benches/bench.rs

```diff
@@ -44,7 +44,7 @@ pub fn http_requests(c: &mut criterion::Criterion) {
 	let server_addr = block_on(rx_addr).unwrap();
 	let client = Arc::new(HttpClient::new(&format!("http://{}", server_addr), HttpConfig::default()).unwrap());
 
-	c.bench_function("synchronous http round trip", |b| {
+	c.bench_function("synchronous_http_round_trip", |b| {
```

Comment:
took me too much time to identify that https://github.com/rhysd/github-action-benchmark/blob/master/src/extract.ts#L181 doesn't support spaces in benchmark names ^^

Reply:
<insert-joke-about-regex-and-now-you-have-two-problems/>